docs/content/documentation/systems/Executioner/TimeStepper/index.md

```diff
@@ -1,6 +1,14 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # TimeStepper System
+
+The method used to calculate the size of the time steps is controlled by the
```

Comment:
The method used to calculate the size of the time steps is controlled by the `TimeStepper` block. There are a number of types of `TimeStepper` available, each of which controls the time stepping in different ways, including using fixed time stepping, time stepping based on a function, or adaptive time stepping.

(just strike the second paragraph)